src/services/notification.js

```diff
@@ -320,6 +320,20 @@ class Notification extends EventEmitter {
   async passwordRecovery(payload) {
     return this.sendMail('passwordRecovery', payload);
   }
+
+  /**
+   * Handles notification for password update successful
+   *
+   * @function
+   *
+   * @param {Array} payload - An Object that contains the user email
+   *
```

Comment:
Please, why is there more than one line space before the return definition in the jsdocs of this function?

Reply:
Thanks for catching that. It is fixed now.